main.py

```diff
@@ -106,7 +103,8 @@ def del_memes(context):
         # lad_bot.edit_message_text(chat_id=meme_sent[2], message_id=meme_sent[1].message_id, text=edited_text)
 
     del MEMES_SENT[:]
-    print(f"{number} meme(s) deleted.")
+    if number != 0:
+        print("Memes deleted.")
```

Comment:
Is this intentional or do I just not get it? Shouldn't it be:
```suggestion
    if not MEMES_SENT:
        print("Memes deleted.")
```